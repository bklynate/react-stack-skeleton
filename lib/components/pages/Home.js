'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.default = Home;

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

function _objectWithoutProperties(obj, keys) { var target = {}; for (var i in obj) { if (keys.indexOf(i) >= 0) continue; if (!Object.prototype.hasOwnProperty.call(obj, i)) continue; target[i] = obj[i]; } return target; }

/**
 * The home page of the website.
 */
function Home(_ref) {
  var otherProps = _objectWithoutProperties(_ref, []);

  return _react2.default.createElement(
    'div',
    otherProps,
    'Home Page'
  );
}