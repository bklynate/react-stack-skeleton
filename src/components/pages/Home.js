/* @flow */
import React from 'react';

type Props = {

};

/**
 * The home page of the website.
 */
export default function Home({ ...otherProps }: Props): React.Element<*> {
  return (
    <div {...otherProps}>
      Home Page
    </div>
  );
}
