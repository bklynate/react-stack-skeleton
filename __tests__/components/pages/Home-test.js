import React from 'react';
import renderer from 'react-test-renderer';

import Home from '../../../src/components/pages/Home';

test('The Home component displays the home page.', () => {
  const component = renderer.create(
    <Home />
  );
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
