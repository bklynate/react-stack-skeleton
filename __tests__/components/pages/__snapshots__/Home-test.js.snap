exports[`test The Home component displays the home page. 1`] = `
<div>
  Home Page
</div>
`;
